kong/init.lua

```diff
@@ -514,7 +514,8 @@ function Kong.ssl_certificate()
 
   runloop.certificate.before(mock_ctx)
 
-  local ok, err = runloop.update_plugins_iterator()
+  local timeout = runloop.get_rebuild_timeout()
+  local ok, err = runloop.rebuild_plugins_iterator(timeout)
```

Comment:
Same here.

Reply:
Answered in other thread, closing this one to keep the conversation focused.